azure-devops/golang/stages/40-delivery/dynamic-go-arm.yaml

```diff
@@ -26,6 +26,10 @@ stages:
               cacheHitVar: 'AZURE_DEPLOY_CACHE_HIT'
```

Comment:
I recommend renaming these files. My suggestion would be:
```
40-delivery/arm-dynamic.yaml (files inside stages)
40-delivery/arm-fixed.yaml

50-deployment/arm-dynamic.yaml
50-deployment/arm-fixed.yaml

golang/arm-dynamic.yaml (files in the root)
golang/arm-fixed.yaml
```
And also, move the templates to the folder "golang/abstracts". Let's avoid repeating words like "go". We know it's go because we are inside the golang directory inside the Azure DevOps directory.